### DISCUSSION TITLE
Updating Active Project Property via API not working

### DISCUSSION THREAD

**Discussion by KyleBaxter9** [DependencyTrack/dependency-track#1936] [Q&A] [answered]
When trying to send a request to update the active property of a project in dtrack I am getting 'uncaught server error' maybe my syntax is off however I assume from the swagger.json documentation on this that I just send a post to the endpoint: project/UUID/property with my body which is {"active": "false"} but this doesn't seem to be working for me.

Any suggestions or does this endpoint not update these properties?

**Comment by nscuro** [comment, marked as answer]
The `active` field is not a property. Properties are generic key-value pairs that can be used in addition to the predefined fields of projects.

As of DT v4.5.0, you can `PATCH` a project to only update certain fields, e.g.:

```
curl -X PATCH -H 'Content-Type: application/json' -d '{"active": false}' https://dtrack.example.com/api/v1/project/a53c7e02-d0f6-4502-a0e7-88e8f18b48f7

**Reply by KyleBaxter9** [reply]
Yeah I realised this after I checked through the changlog issues, thank you.
